lab_6_pipeline/pipeline.py

```diff
@@ -144,17 +241,63 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._stemmer = Mystem()
```

Comment:
conceptually wrong name. we use it as analyzer and lemmatizer, not as a stemmer. btw, do you know the difference?

Reply:
Fixed. Yes, I do